ppa/archive/gale.py

```diff
@@ -192,14 +208,19 @@ def get_item_pages(self, item_id, gale_record=None):
         # iterate through the pages in the response
         for page in gale_record["pageResponse"]["pages"]:
             page_number = page["pageNumber"]
-            # page label (original page number) should be set in folioNumber,
-            # but is not set for all volumes; fallback to page number
-            # converted to integer to drop leading zeroes
-            page_label = page.get("folioNumber", int(page_number))
+            tags = []
+            try: 
+                ocr_text = get_local_ocr(item_id, page_number)
+                tags = ["local_ocr"]
+            except FileNotFoundError as e:
+                ocr_text = page.get("ocrText")  # some pages have no text
+                logger.warning(f'Local OCR not found for {item_id} {page_number}')
+
             info = {
                 "page_id": page_number,
-                "content": page.get("ocrText"),  # some pages have no text
-                "label": page_label,
+                "content": ocr_text,
+                "label": page.get("folioNumber"),
```

Comment:
This one also seems worth commenting (thanks again for catching this!). Maybe something like this?

```suggestion
                # don't set label when we don't have one, so we get fall-back behavior in common page indexing code
                "label": page.get("folioNumber"),
```